home/.config/home-manager/home.nix

```diff
@@ -20,6 +20,31 @@
   # changes in each release.
   home.stateVersion = "23.05";
 
+  home = {
+    sessionVariables = {
+      INPUTRC = "${config.home.sessionVariables.XDG_CONFIG_HOME}/readline/inputrc";
+      EDITOR = "code -w";
+      VISUAL = "nano";
+      PAGER = "less";
+
+      # - You can check the candidates in `locale -a`
+      # - pkgs.glibc installs many candidates
```

Comment:
but it does not support darwin 😢  

```
       error: Package ‘glibc-2.37-8’ in /nix/store/m2grimlkdh0gr1h97frpgrxdgfgp3b8v-nixpkgs/nixpkgs/pkgs/development/libraries/glibc/default.nix:173 is not available on the requested hostPlatform:
         hostPlatform.config = "x86_64-apple-darwin"
         package.meta.platforms = [
           "aarch64-linux"
           "armv5tel-linux"
           "armv6l-linux"
           "armv7a-linux"
           "armv7l-linux"
           "i686-linux"
           "loongarch64-linux"
           "m68k-linux"
           "microblaze-linux"
           "microblazeel-linux"
           "mips-linux"
           "mips64-linux"
           "mips64el-linux"
           "mipsel-linux"
           "powerpc64-linux"
           "powerpc64le-linux"
           "riscv32-linux"
           "riscv64-linux"
           "s390-linux"
           "s390x-linux"
           "x86_64-linux"
         ]
         package.meta.badPlatforms = [ ]
       , refusing to evaluate.

       a) To temporarily allow packages that are unsupported for this system, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1
```

Reply:
Fixed in https://github.com/kachick/dotfiles/pull/201/commits/4400922557a8479f09f45e261b56e5a3490ab085